gui/src/app/components/copy-to-clipboard/copy-to-clipboard.js

```diff
@@ -19,8 +19,10 @@ class CopyToClipboard extends React.Component {
   componentDidMount() {
     var node = ReactDOM.findDOMNode(this).querySelector('button');
     this.clipboard = new Clipboard(node, {
-      target(trigger) {
-        return trigger.previousElementSibling;
+      target: trigger => {
+        this.props.analytics.addCategory(this).sendEvent(
+          this.props.analytics.CLICK, {label: `entity: ${this.props.value}`});
```

Comment:
Entity is not clear here. Also could the label describe where the input is and the value. This would help to determine which ones being used.